submissions/HelenGreent/task_html_css_popup/index.html

```diff
@@ -0,0 +1,185 @@
+                      </li>
+                      <li class="popup__list-item">
+                        <a class="popup__link" href="#">
+                          <img
+                            class="popup__link-icon"
+                            src="./images/maps.png"
+                            alt="maps"
+                          />
+                          <span class="popup__link-title">Maps</span>
+                        </a>
+                      </li>
+                      <li class="popup__list-item">
+                        <a class="popup__link" href="#">
+                          <img
+                            class="popup__link-icon"
+                            src="./images/photos.png"
+                            alt="photo"
+                          />
+                          <span class="popup__link-title">Photos</span>
+                        </a>
+                      </li>
+                    </ul>
+                    <div>
+                      <div class="add-popup">
+                        <input
+                          class="add-popup__control"
+                          type="checkbox"
+                          id="popup-more"
+                        />
+                        <label class="add-popup__toggler" for="popup-more">
+                          <span class="add-popup__title">More</span>
+                        </label>
+                        <div class="add-popup__content">
+                          <ul class="popup__list">
+                            <li class="popup__list-item">
+                              <a class="popup__link" href="#">
+                                <img
+                                  class="popup__link-icon"
+                                  src="./images/favicon.png"
+                                  alt="kottans"
+                                />
+                                <span class="popup__link-title">Kottans</span>
+                              </a>
+                            </li>
+                            <li class="popup__list-item">
+                              <a class="popup__link" href="#">
+                                <img
+                                  class="popup__link-icon"
+                                  src="./images/collection.png"
+                                  alt="collection"
+                                />
+                                <span class="popup__link-title"
+                                  >Collection</span
+                                >
+                              </a>
+                            </li>
+                            <li class="popup__list-item">
+                              <a class="popup__link" href="#">
+                                <img
+                                  class="popup__link-icon"
+                                  src="./images/documents.png"
+                                  alt="doc"
```

Comment:
Please read about `alt` attribute and make corresponding changes.
The alt attribute provides alternative information for an image if a user for some reason cannot view it (because of slow connection, an error in the src attribute, or if the user uses a screen reader).
https://www.w3.org/QA/Tips/altAttribute